<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>800</width>
				<height>600</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>VisionBoard Server</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<widget class="QPushButton" name="startServer">
				<property name="geometry">
					<rect>
						<x>300</x>
						<y>390</y>
						<width>111</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Start Server</string>
				</property>
			</widget>
			<widget class="QPushButton" name="stopServer">
				<property name="geometry">
					<rect>
						<x>420</x>
						<y>390</y>
						<width>111</width>
						<height>41</height>
					</rect>
				</property>
				<property name="text">
					<string>Stop Server</string>
				</property>
			</widget>
			<widget class="QListWidget" name="displaysClientsList">
				<property name="geometry">
					<rect>
						<x>530</x>
						<y>40</y>
						<width>256</width>
						<height>341</height>
					</rect>
				</property>
			</widget>
			<widget class="QTextEdit" name="clientLogs">
				<property name="geometry">
					<rect>
						<x>20</x>
						<y>40</y>
						<width>501</width>
						<height>341</height>
					</rect>
				</property>
			</widget>
			<widget class="QLabel" name="clientCountLabel">
				<property name="geometry">
					<rect>
						<x>530</x>
						<y>390</y>
						<width>256</width>
						<height>21</height>
					</rect>
				</property>
				<property name="text">
					<string>Connected Clients: 0</string>
				</property>
			</widget>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>800</width>
					<height>22</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
</ui>